web/src/javascript/drawMap.js

```diff
@@ -58,13 +60,53 @@ export function drawMiniMap() {
 
   drawMiniMapminiMapScreenPos();
 
-  screenContext.drawImage(miniMapCanvas, miniMapPosX,0);
+  screenContext.drawImage(miniMapCanvas, miniMapPosX, 0);
   screenCanvas.style.background = "#000";
 }
 
-function drawMiniMapminiMapScreenPos(){
-  miniMapContext.strokeStyle="#fff";
-  var xMiniMapPos = miniMapWidth/mapWidth*xScreenPosOnMap;
-  var yMiniMapPos = miniMapHeight/mapHeight*yScreenPosOnMap;
-  miniMapContext.strokeRect(xMiniMapPos,yMiniMapPos,miniMapScreenPosWidth,miniMapScreenPosHeight);
+function drawMiniMapminiMapScreenPos() {
+  miniMapContext.strokeStyle = "#fff";
+  var xMiniMapPos = miniMapWidth / mapWidth * xScreenPosOnMap;
+  var yMiniMapPos = miniMapHeight / mapHeight * yScreenPosOnMap;
+  miniMapContext.strokeRect(xMiniMapPos, yMiniMapPos, miniMapScreenPosWidth, miniMapScreenPosHeight);
+}
+
+function changeScreenPos(mousePos) {
+  var miniMapPos = {
+    x : (mousePos.x - miniMapPosX) - (miniMapScreenPosWidth / 2),
+    y : mousePos.y - (miniMapScreenPosHeight / 2)
+  }
```

Comment:
Is this function to change what we see on the main screen based on where we click on the minimap? I'm eager to see this live!! 😄 

Also, is it normal it is called `drawMiniMapminiMapScreenPos` with two times minimap in the name?